shared/js/background/companies.js

```diff
@@ -87,8 +88,7 @@ const Companies = (() => {
             totalPagesWithTrackers = 0
             lastStatsResetDate = Date.now()
             Companies.syncToStorage()
-            const resetDate = Companies.getLastResetDate()
-            browserWrapper.notifyPopup({ didResetTrackersData: resetDate })
+            postPopupMessage({ messageType: 'didResetTrackersData' })
```

Comment:
_Shane confirmed the `resetDate` isn't used at all, and in fact including it will throw a type error now._